code/track.ss

```diff
@@ -9,38 +9,8 @@
       (delete-file config.json))
     (with-output-to-file config.json
       (lambda ()
-	(json-write (process-config))))))
-
-;; preprocess nodes in config. currently removes exercises marked wip
-(define (process-config)
-  (map (lambda (x)
-         (if (not (eq? (car x) 'exercises))
-             x
-             `(exercises . ,(remp (lambda (exercise)
-                                    (memq 'wip (map car exercise)))
-                                  (cdr x)))))
-       track-config))
-
-;; Check problem's entry in config for uuid and existence.
-(define (check-config-for problem)
```

Comment:
I didn't mean to remove this.  Let me see how to get it back.

Reply:
Maybe to follow some of your advice it would be better to fix the trailing whitespace/formatting in a separate request? It makes it harder to see what the substantial additions are